src/main/java/com/marklogic/client/workspaces/WorkspacesImporter.java

```diff
@@ -0,0 +1,12 @@
+package com.marklogic.client.workspaces;
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Set;
+
+import com.marklogic.client.DatabaseClient;
+
+public interface WorkspacesImporter {
+
+	Set<File> importWorkspaces(Path path, DatabaseClient client, String user);
```

Comment:
Similar question as above - what does the user refer to here?

Following my example interface above, I think the simplest thing here would be:

    Set<File> importWorkspaces(DatabaseClient client, String... workspaceNames)

We can assume a location of "./data/qconsole/workspaces". And if workspaceNames is empty, all workspaces are loaded.